### PR TITLE
new workflow: Allow to add a product::component in list of components

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: auto_nag/scripts/workflow/p3_p4_p5.py [hunk 1]
```diff
@@ -28,13 +28,19 @@ def has_product_component(self):
     def columns(self):
         return ['component', 'id', 'summary']
 
+    def handle_bug(self, bug, data):
```
Comment (sylvestre) [inline auto_nag/scripts/workflow/p3_p4_p5.py]: maybe move that stuff in utils?

File: auto_nag/utils.py [hunk 2]
```diff
@@ -411,3 +411,19 @@ def get_info_from_hg(json):
 
 def bz_ignore_case(s):
     return '[' + ']['.join(c + c.upper() for c in s) + ']'
+
+
+def check_pc(data, bug):
```
Comment (sylvestre) [inline auto_nag/utils.py]: check_product_component?

File: auto_nag/scripts/workflow/p3_p4_p5.py (hunk 1 above)
Comment (calixteman) [reply auto_nag/scripts/workflow/p3_p4_p5.py]: handle_bug is defined in bzcleaner class and so we need to overwrite it to filter the bugs

File: auto_nag/utils.py (hunk 2 above)
Comment (calixteman) [reply auto_nag/utils.py]: done

**Pull request merged**